woocommerce/woocommerce
# Add support for 6 new countries in WCPay

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



This PR adds onboarding support for 6 new countries in WCPay. [Another PR](https://github.com/woocommerce/woocommerce/pull/36906) with the same goal was added a few months ago.

**This PR targets the next version of WC 7.8.0**



### How to test the changes in this Pull Request:



To trigger the OBW fallback, disable marketplace suggestions:
Settings > [WooCommere.com](http://woocommere.com/) > untick Display suggestions within WooCommerce

 1. Get this PR locally.
 2. Change your store location to one of the countries in this list (eg. Romania, Croatia, etc.).
 3. Trigger the WooCommerce onboarding wizard.
 4. Make sure WC Payments is showing as the suggested payment gateway in the "Set up payments" step.
